contracts/test/spaces/integration/createSpace/createSpace.t.sol

```diff
@@ -36,11 +36,14 @@ contract Integration_CreateSpace is
     spaceArchitect = Architect(spaceFactory);
   }
 
-  function test_createEveryoneSpace(string memory spaceId) external {
+  function test_fuzz_createEveryoneSpace(
+    string memory spaceId,
+    address founder,
+    address user
+  ) external {
     vm.assume(bytes(spaceId).length > 2 && bytes(spaceId).length < 100);
-
-    address founder = _randomAddress();
-    address user = _randomAddress();
+    // founder must be EOA or implement `onERC721Received`
```

Comment:
Does this comment match the vm.assume? If it implements onERC721Recevied the code length would be >0, right?

Reply:
It could be phrased better. It's either and EOA or a contract implemented `onERC721Received`. It's easier to fuzz EOAs without code.